ray_data_eval/image_training/image_loader_microbenchmark.py

```diff
@@ -232,6 +233,16 @@ def decode_image_crop_and_flip(row):
     return {"image": np.array(transform(row["image"]))}
 
 
+def _collate_fn_arr_pil_images_to_tensor(batch: np.ndarray) -> torch.Tensor:
```

Comment:
Add a docstring saying that this function expects the input to be an array of PIL.Images and outputs blah blah blah